script/openqa-clone-job

```diff
@@ -133,6 +133,10 @@ are cloned. Use C<0> to denote infinity.
 A shortcut for C<--skip-download --from HOST --host HOST> to clone a job within
 a local or remote instance.
 
+=item B<--repeat> N
+
+Do the same operation N times. Allowed values are from 1 to 99
```

Comment:
Why that limit? I see good use cases for much higher numbers and besides no one stops you from calling the command with repeat like 1000 times except for a rate limit on the server which is out of scope for the addition to the client scripts. I would just not limit here at all

Reply:
> Why that limit? I see good use cases for much higher numbers, and besides, no one stops you from calling the command with repeat like 1000 times except for a rate limit on the server which is out of scope for the addition to the client scripts. I would not limit myself here at all

This basically invites a mistake of scheduling 1k jobs :D and more

Reply:
Still any limit is arbitrary and I see valid use cases to schedule more than thousand jobs. Please just don't limit here at all. openqa-clone-job is just not the place for it to decide. If you really want to prevent mistakes then make it a soft-limit of let's say 300 and allow to override with another parameter, e.g. "--force-repeat". I would just go with no limit for now. After all the iteration would still be done client-side so scheduling 1k jobs takes possibly some minutes and the user can still abort at any time with ^C

Reply:
If you don't limit at all, then make also sure the code for appending the index can handle a variable number of digit. The code in its current form would not be able to handle that because it replaces the previous index using the number of digits of the current index which would not work when the number of digits actually differs.

Reply:
I'd second point from @Martchus; of course, it's possible to handle variable digit numbers, but IMHO It's not really worth it because adding more digits makes the code more complicated for the (small?) percentage of use cases when is really needed to clone a job over 1000 times, which BTW can be accomplished if the user just press up arrow and repeat the previous command. 
So my proposal is to keep it to 3 digits and see if there is a need, we can always improve on that.

Reply:
I regularly schedule 300 jobs at once and if you added a hard limit you'd only force me to make mistakes by having to use shell syntax to work around it ;-)

Reply:
Also, thanks for adding this. I'll definitely use it

Reply:
Thanks, 300 is well under the current (updated) 999 limit, so seems it's good also for you :)